### DISCUSSION TITLE
Free type mode

### DISCUSSION THREAD

**Discussion by frogfrogfrogfrogfrog** [monkeytypegame/monkeytype#801] [Ideas] [answered]
A mode where you can freely type whatever you want with your keyboard layout. It could have a character limit as well. I guess this already exists with the custom text generation but whenever your making the custom text generation you have to use the native keyboard layout.

**Comment by frogfrogfrogfrogfrog** [comment, marked as answer]
Okay I just found out that zen mode is about what I'm looking for. So this discussion is useless.

**Reply by ghost** [reply]
How do you use zen mode?

**Reply by frogfrogfrogfrogfrog** [reply]
You can't currently. It's still in development but I decided this discussion is useless since it's already being developed

**Reply by ghost** [reply]
It's added!

**Reply by frogfrogfrogfrogfrog** [reply]
woohoo lets goooooo
